sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateClient.java

```diff
@@ -1197,7 +1062,7 @@ public KeyVaultCertificate mergeCertificate(MergeCertificateOptions mergeCertifi
      * @throws HttpRequestException when the {@code importOptions} are invalid.
      * @return the {@link KeyVaultCertificate imported certificate}.
      */
-    public KeyVaultCertificate importCertificate(CertificateImportOptions importOptions) {
+    public KeyVaultCertificate importCertificate(ImportCertificateOptions importOptions) {
```

Comment:
Following `mergeCertificateOptions` above, shouldn't the param here and below be `importCertificateOptions`?